digitalocean_plugin/tests/test_security.py

```diff
@@ -0,0 +1,278 @@
+########
```

Comment:
Include a description in the doc string at beginning of each test explaining what the test does and what different outcomes would be under different circumstances.